R/animint.R

```diff
@@ -130,11 +133,12 @@ parsePlot <- function(meta){
     ## Data now contains redundant columns with fill, alpha, colour etc.
     ## Remove from data if they have a single unique value and
     ## are NOT used in mapping to reduce tsv file size
+    redundant.cols <- c("fill", "shape", "color", "colour", "stroke", "alpha",
+                        "linetype", "size", "angle", "vjust", "family",
+                        "fontface", "lineheight", "anchor")
```

Comment:
@faizan-khan-iit is there a reason why we need to list all the column names in the R code, rather than just detecting them based on the data?